certification/internal/shell/base_on_ubi.go

```diff
@@ -1,21 +1,28 @@
 package shell
 
 import (
+	"fmt"
 	"strings"
 
 	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification"
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification/errors"
 	"github.com/redhat-openshift-ecosystem/openshift-preflight/cli"
 	log "github.com/sirupsen/logrus"
 )
 
 type BaseOnUBICheck struct{}
 
 func (p *BaseOnUBICheck) Validate(image string) (bool, error) {
-	podmanEngine := PodmanCLIEngine{}
-	return p.validate(podmanEngine, image)
+	lines, err := p.getDataToValidate(image)
+	if err != nil {
+		log.Debugf("Stdout: %s", lines)
+		return false, fmt.Errorf("%w: %s", errors.ErrFormattingResults, err)
```

Comment:
Hmm, should we use a different error here? Under the premise that "Results" as a concept refers to the validation results in its entirety?

Reply:
Sure. I'll change that. Probably just a case of me being lazy.